arch/xtensa/src/common/xtensa_int_handlers.S

```diff
@@ -312,6 +312,7 @@ _xtensa_level1_handler:
 
 	s32i	a2, sp, (4 * REG_A2)
 	mov		a2, sp							      /* Address of state save on stack */
+	s32i	a3,  a2, (4 * REG_A3)
```

Comment:
```suggestion
	s32i	a3, a2, (4 * REG_A3)
```
nit: double whitespace

Reply:
By the way, since the `SP` regsiter is still unchanged at this point, isn't it more intuitive to use it as the base address?